### ISSUE TITLE
Clip is totally transparent when crossfadeout effect starts with time resized clip.

### ISSUE THREAD

**Issue by obiwankennedy** [Zulko/moviepy#1196]
This is a small example where the issue occurs. I may do something wrong but I wondering how to get this to work.
```
#!/usr/bin/python3.8
# -*- coding: utf-8 -*-

from moviepy.editor import *


clip1 = ColorClip((200,200),color=(255,0,0),duration=10)
clip2 = ColorClip((200,200),color=(0,255,0),duration=10)

clip1 = clip1.set_position((860,440))
clip2 = clip2.set_position((860,440))

clip1 = clip1.set_end(10).crossfadeout(1)
clip2 = clip2.set_start(clip1.end - 1).crossfadein(1)
clip2 = clip2.set_end(20).crossfadeout(1)

# Those two lines cause the issue
clip1 = clip1.resize(lambda t: 1 + 0.05 * t)
clip2 = clip2.resize(lambda t: 1 + 0.05 * t)

compo = CompositeVideoClip([clip1,clip2], size=[1920,1080], use_bgclip=False, bg_color=(0,0,0))
compo.write_videofile("example.mp4", fps=30)
```

### Expected Behavior
I want my clip to get bigger and to fadeout to let the next clip appears.

### Actual Behavior
If I try to resize the clip given the time, when the fadeout should start, the clip disappears.
If I comment out the resize part, the clips are shown as I expect but, I really need to resize them.

### Steps to Reproduce the Problem
Just run my example, try with or without the resize part.

### Specifications

  - Python Version: 3.8
  - Moviepy Version: v1.0.3
  - Platform Name: Ubuntu
  - Platform Version: 18.04

Thank you!
Regards.

**Comment by tburrows13** [comment]
Hello, thanks for the report. I can confirm that I can replicate this issue.

**Comment by obiwankennedy** [comment]
Is there a workaround I can use ? or it is a real bug ?

**Comment by tburrows13** [comment]
Hi, it is definitely a bug, which I've fixed in #1200. Are you able to edit the file `resize.py` in your MoviePy installation to match my changes?

If you can't then I can try to push a v2.0.0.dev1 release to pip which would include this change and many others that will be in the final v2.0.0 release.

**Comment by obiwankennedy** [comment]
It is fine, I applied the patch on my own install, so take your time to publish the next release.
Thank you. The result is great! 
Should I close the ticket or do you want to close it when you release the next version ?

**Comment by tburrows13** [comment]
It will get closed when #1200 is merged into master :). Good to know that it worked!
